sematic/db/models/artifact.py

```diff
@@ -21,15 +22,17 @@ class Artifact(Base, HasOrganizationMixin, JSONEncodableMixin):
 
     __tablename__ = "artifacts"
 
-    id: str = Column(types.String(), primary_key=True)
-    json_summary: str = Column(types.JSON(), nullable=False, info={JSON_KEY: True})
-    type_serialization: str = Column(
+    id: Mapped[str] = mapped_column(types.String(), primary_key=True)
```

Comment:
SQLAlchemy 2.0 starts to introspect on type hints for columns. This is the minimal-change recommended by the migration guide to keep existing type hints around.